exportify.js

```diff
@@ -185,16 +185,21 @@ let PlaylistExporter = {
 		// "returns a single Promise that resolves when all of the promises passed as an iterable have resolved"
 		// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all
 		let artist_ids = new Set()
+		let album_ids = new Set()
 		let data_promise = Promise.all(requests).then(responses => { // Gather all the data from the responses in a table.
 			return responses.map(response => { // apply to all responses
 				return response.items.map(song => { // apply to all songs in each response
 					// Safety check! If there are artists listed and they have non-null identifier, add them to the set
 					song.track?.artists?.forEach(a => { if (a && a.id) { artist_ids.add(a.id) } });
+					 // Collect album IDs
+					 if (song.track?.album && song.track.album.id) {
```

Comment:
I'm probably gonna come back and one-line this like I do with adding to the `artist_ids` set.